chain/gen/genesis/genesis.go

```diff
@@ -215,6 +216,17 @@ func MakeInitialStateTree(ctx context.Context, bs bstore.Blockstore, template ge
 		return nil, nil, xerrors.Errorf("set verified registry actor: %w", err)
 	}
 
+	// Create datacap actor
```

Comment:
Can you make a genesis block starting from v17, and run a devnet with it? it's a little hard to have confidence in correctness of stuff like this without that.

(The itests also exercise this path, but let's be sure)